subworkflows/nf-core/bam_ngscheckmate/nextflow.config

```diff
@@ -0,0 +1,13 @@
+// IMPORTANT: Add this configuration to your modules.config
+
+process {
+    withName: ".*BAM_NGSCHECKMATE:BCFTOOLS_MPILEUP" {
+        ext.args2 = '--no-version --ploidy 1 -c'
+        ext.args3 = '--no-version'
+    }
+
+    withName: ".*BAM_NGSCHECKMATE:NGSCHECKMATE_NCM" {
+        ext.args = '-V'
+    }
+
+}
```

Comment:
😬 this one is not loaded i hope

Reply:
not at the moment

Reply:
Now we do

Reply:
But it's the updated, so it's the one we used to use, nothing should change there

Reply:
I don't like it. it gets to hard to find anything. In any case it then also needs closures

Reply:
there is a ngscheckmate cnfig though in conf/ that has these values, so I don't think we need to include it

Reply:
on dev anyways, and looks like that one needs closures too.

Reply:
Kept the old one as it was then